test/uk/gov/hmrc/helptosavefrontend/controllers/IntroductionControllerSpec.scala

```diff
@@ -31,7 +31,7 @@ import scala.concurrent.Future
 
 class IntroductionControllerSpec extends AuthSupport {
 
-  implicit val timeout = Timeout(5, SECONDS)
+  implicit val timeout: Timeout = Timeout(5, SECONDS)
```

Comment:
can we move this timeout to base class TestSupport or AuthSupport instead of declaring in each spec ?